packages/components/src/avatar/tests/chromatic/Avatar.chroma.jsx

```diff
@@ -2,98 +2,96 @@ import { Avatar } from "@components/avatar";
-                <Avatar src={Person} name="Neil Armstrong" />
-            </Div>
-        </Stack>
-    )
-    .add("styling", () =>
-        <Inline>
-            <Avatar name="Sally Ride" border="warning-7" />
-            <Avatar name="Sally Ride" className="border-red" />
-            <Avatar name="Sally Ride" style={{ border: "1px solid red" }} />
-        </Inline>
-    );
+export const LocalImage = () => <Inline alignY="center">
+    <Avatar src={Person} size="2xs" name="Neil Armstrong" />
+    <Avatar src={Person} size="xs" name="Neil Armstrong" />
+    <Avatar src={Person} size="sm" name="Neil Armstrong" />
+    <Avatar src={Person} name="Neil Armstrong" />
+    <Avatar src={Person} size="lg" name="Neil Armstrong" />
+    <Avatar src={Person} size="xl" name="Neil Armstrong" />
+    <Avatar src={Person} size="2xl" name="Neil Armstrong" />
+</Inline>;
+
+export const RemoteImage = () => <Avatar size="2xl" name="Neil Armstrong" src="https://randomuser.me/api/portraits/men/10.jpg" />;
+
+export const FailingRemoteSrc = () => <Inline alignY="center">
+    <Avatar size="2xs" src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar size="xs" src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar size="sm" src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar size="lg" src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar size="xl" src="https://www.google.com" name="Neil Armstrong" />
+    <Avatar size="2xl" src="https://www.google.com" name="Neil Armstrong" />
+</Inline>;
+
+export const Initials = () => <Stack>
+    <Inline alignY="center">
+        <Avatar size="2xs" name="Neil Armstrong" />
+        <Avatar size="xs" name="Neil Armstrong" />
+        <Avatar size="sm" name="Neil Armstrong" />
+        <Avatar name="Neil Armstrong" />
+        <Avatar size="lg" name="Neil Armstrong" />
+        <Avatar size="xl" name="Neil Armstrong" />
+        <Avatar size="2xl" name="Neil Armstrong" />
+    </Inline>
+    <Inline>
+        <Avatar name="Sally Ride" />
+        <Avatar name="Alan Shepard" />
+        <Avatar name="Chris Hadfield" />
+        <Avatar name="Christa McAuliffe" />
+    </Inline>
+</Stack>;
+
+export const EmptySrc = () => <Inline alignY="center">
+    <Avatar src="" size="2xs" name="Neil Armstrong" />
+    <Avatar src="" size="xs" name="Neil Armstrong" />
+    <Avatar src="" size="sm" name="Neil Armstrong" />
+    <Avatar src="" name="Neil Armstrong" />
+    <Avatar src="" size="lg" name="Neil Armstrong" />
+    <Avatar src="" size="xl" name="Neil Armstrong" />
+    <Avatar src="" size="2xl" name="Neil Armstrong" />
+</Inline>;
+
+export const Zoom = () => <Stack>
+    <Div className="zoom-in">
+        <Avatar src={Person} name="Neil Armstrong" />
+    </Div>
+    <Div className="zoom-out">
+        <Avatar src={Person} name="Neil Armstrong" />
+    </Div>
+</Stack>;
+
+export const Styling = () => <Inline>
```

Comment:
Not a big deal. more of a personal preference but I am not a big fan of having the opening element on the same line as the dclaration as it's then not aligned with the closing element.

I would rather do

```jsx
export const Zoom = () => (
   <Stack>
       <Div className="zoom-in">
           <Avatar src={Person} name="Neil Armstrong" />
       </Div>
       <Div className="zoom-out">
           <Avatar src={Person} name="Neil Armstrong" />
       </Div>
   </Stack>
);
```

Reply:
Sure I feel the same, the first stories I did were like this as I copied pasted from an example ;) will fix now.

Reply:
Parenthesis are non necessary, do you prefer to have them?

Reply:
My personal preference is to have them but it’s not a big deal if you prefer not to